src/app/grid/grid-summary-formatter/grid-summary-formatter.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="grid__wrapper">
+    <igx-grid igxPreventDocumentScroll #grid1 [data]="data" [autoGenerate]="false" height="600px" width="100%"
+        [allowFiltering]='true' filterMode="excelStyleFilter">
+        <igx-column field="ProductName" header="Product Name" [sortable]="true" [disableHiding]="true" [dataType]="'string'">
+        </igx-column>
+        <igx-column field="QuantityPerUnit" header="Quantity Per Unit" [sortable]="true" [disableHiding]="true" [dataType]="'string'">
+        </igx-column>
+        <igx-column field="UnitPrice" header="Unit Price Category" [sortable]="true" [disableHiding]="true" dataType="string" [formatter]="formatPrice"
+            [hasSummary]="true">
```

Comment:
It is better to remove the summaries on the other columns.
```suggestion
        <igx-column field="UnitPrice" header="Unit Price Category" [sortable]="true" [disableHiding]="true" dataType="string" [formatter]="formatPrice">
```